gcloud-java-dns/src/main/java/com/google/gcloud/dns/ManagedZoneInfo.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.gcloud.dns;
+
+/**
+ * todo(mderka): Implement.
+ * todo(mderka): Add documentation.
+ */
+public class ManagedZoneInfo {
```

Comment:
This class is incomplete and without a test. I think you could have done with out it, but especially not
needed if `DnsRecord` remove reference to parent.

Please remove this file from this PR and add later (with test).